src/cdk/portal/portal.spec.ts

```diff
@@ -299,15 +299,15 @@ describe('Portals', () => {
       fixture.detectChanges();
 
       // Attach the TemplatePortal.
-      testAppComponent.portalWithBinding.attach(host);
+      testAppComponent.portalWithBinding.attach(host, { $implicit: { status: 'fresh' } });
```

Comment:
Actually the tests are limited since there are no are no "low-level" tests for `TemplatePortal` with a `PortalHostDirective` (there are some for `ComponentPortal`)

I'v added 2 tests:
  1. **should load a template into the portal**
Check basic projection of programatically created `TemplatePortal`

  2. **should project template context bindings in the portal**
Check project of programatically created `TemplatePortal` with 4 internal `expects`
       - No context
       - Context through `attach()` method
       - Context through **constructor**
       - Context through **constructor** and through `attach()` method - the latter must take precedence.